### DISCUSSION TITLE
Trouble running helloworld

### DISCUSSION THREAD

**Discussion by gvidaver** [grpc/grpc-web#1355] [Q&A - Errors / Debugging Help]
I'm trying to run the helloworld example on a mac but I'm getting:

```
::1 - - [23/Aug/2023 15:45:03] "GET /dist/main.js HTTP/1.1" 200 -
::1 - - [23/Aug/2023 15:45:03] code 404, message File not found
::1 - - [23/Aug/2023 15:45:03] "GET /favicon.ico HTTP/1.1" 404 -
::1 - - [23/Aug/2023 15:45:13] "GET / HTTP/1.1" 304 -
```

All the services are up:

```
(base) xxx@yyy helloworld % netstat -an | grep LISTEN
tcp46      0      0  *.8081                 *.*                    LISTEN     
tcp4       0      0  *.9090                 *.*                    LISTEN   
```

And in the browser:
<img width="767" alt="Screenshot 2023-08-23 at 4 20 20 PM" src="https://github.com/grpc/grpc-web/assets/4854720/2b8ea7c5-6d39-4c5f-8105-eefc7f5ddd45">

All the rpc files are there:
```
(base) xxx@yyy helloworld % ls -l helloworld*
-rw-r--r--  1 xxx  staff    975 Aug 23 14:17 helloworld.proto
-rw-r--r--@ 1 xxx  staff   4970 Aug 23 15:40 helloworld_grpc_web_pb.js
-rw-r--r--@ 1 xxx  staff  15375 Aug 23 15:40 helloworld_pb.js
```

How should I debug this?
Thanks!

**Comment by gvidaver** [comment]
Sorry - my bad - I didn't follow the Mac/Windows sections in the README faithfully enough. 
After changing the address in the yaml file and running the docker command with the port binding, everything worked fine.

**Reply by sampajano** [reply]
Great to hear thanks for the update! 😃
